### PR TITLE
Fix null pointer exception

### PR DESCRIPTION
**Pull request by RupinderKaurSSB**
Fixed exception when Fetching snr from output when lookup service return null object .

### CODE REVIEW

File: src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java [hunk 1]
```diff
@@ -116,8 +118,13 @@ private PseudoFuncOutput createMappingLogsAndOutput(SidInfo sidInfo, boolean isF
                 PseudoFuncOutput output = PseudoFuncOutput.of(sidInfo.snr());
                 output.addWarning(message);
                 return output;
+            }else{
```
Comment (bjornandre) [inline src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java:121]:
```suggestion
            } else {
```
The code is not properly formatted. The IDE should fix that for you

File: src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java [hunk 2]
```diff
@@ -130,8 +137,14 @@ private PseudoFuncOutput createMappingLogsAndOutput(SidInfo sidInfo, boolean isF
                 output.addWarning(message);
                 return output;
             }
-            return PseudoFuncOutput.of(sidInfo.fnr());
-        }
+            else{
```
Comment (bjornandre) [inline src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java:140]:
```suggestion
            else {
```

**Review by bjornandre — APPROVED** [review]

File: src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java (hunk 2 above)
Comment (RupinderKaurSSB) [reply src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java]: Fixed

File: src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java (hunk 1 above)
Comment (RupinderKaurSSB) [reply src/main/java/no/ssb/dlp/pseudo/service/sid/SidMapper.java]: fixed

**Review by bjornandre — APPROVED** [review]

**Pull request merged**